src/test/java/core/basesyntax/service/FileServiceImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileServiceImplTest {
+    private static final String transactionsFileName = "src" + File.separator + "test"
+            + File.separator + "sources" + File.separator + "inputData.csv";
+    private static final String reportFileName = "src" + File.separator + "test" + File.separator
+            + "sources" + File.separator + "reports.csv";
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

as File.separator you can use simple **/**